### PR TITLE
Move literals to config/locales

### LINKED ISSUE #3536: Move locales to config
Currently all the gobierto-dashboards locales are hardcoded. Move them to its proper location.

### PR DESCRIPTION
**Pull request by Crashillo**
## :v: What does this PR do?
Pull all locales apart, to their respective config files

### CODE REVIEW

File: app/javascript/gobierto_dashboards/webapp/Maker.vue [hunk 1]
```diff
@@ -39,7 +39,7 @@
           <SmallCard
             v-for="({ name }, key) in cards"
             :key="key"
-            :name="name"
+            :name="name()"
```
Comment (ferblape) [inline app/javascript/gobierto_dashboards/webapp/Maker.vue:42]: why is it a function call?

Comment (ferblape) [reply app/javascript/gobierto_dashboards/webapp/Maker.vue:42]: OK, it comes from Widget, right?

File: config/locales/gobierto_dashboards/views/ca.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    changes: Dashboard modificat
+    date: 'Data: '
+    default_title: Títol del dashboard
+    delete: Esborrar
+    instructions: |
+      "⇧ Selecciona un tipus de targeta i inclou-la en l'àrea central ➞
+
+      Després podràs configurar-la."
+    objective: Objectiu
+    percentage: Assoliment
```
Comment (ferblape) [inline config/locales/gobierto_dashboards/views/ca.yml]: Progres?

File: app/javascript/gobierto_dashboards/webapp/Maker.vue (hunk 1 above)
Comment (Crashillo) [reply app/javascript/gobierto_dashboards/webapp/Maker.vue:42]: Exactly. I need to call `I18n.t()` after mounted in order to solve properly the displayed language.

File: config/locales/gobierto_dashboards/views/ca.yml (hunk 2 above)
Comment (Crashillo) [reply config/locales/gobierto_dashboards/views/ca.yml]: _Consecució_, perhaps, to keep consistency with spanish??

Comment (ferblape) [reply config/locales/gobierto_dashboards/views/ca.yml]: ok!

Comment (Crashillo) [reply config/locales/gobierto_dashboards/views/ca.yml]: Done

**Review by jorgeatgu — APPROVED** [review]

**Pull request merged**